core/modules/parsers/wikiparser/rules/html.js

```diff
@@ -60,6 +64,27 @@ exports.parse = function() {
 			var reEnd = new RegExp("(" + reEndString + ")","mg");
 			tag.children = this.parser.parseInlineRun(reEnd,{eatTerminator: true});
 		}
+		tag.end = this.parser.pos;
+		tag.closeTagEnd = tag.end;
+		if (tag.closeTagEnd === tag.openTagEnd) {
+			tag.closeTagStart = tag.end;
+		} else {
+			tag.closeTagStart = tag.end - 1;
+			while (tag.closeTagStart >= tag.start) {
+				var char = this.parser.source[tag.closeTagStart - 1];
+				if (char === '>') {
+					tag.closeTagStart = -1;
+					break;
+				}
+				if (char === '<') {
+					break;
+				}
+				tag.closeTagStart--;
```

Comment:
We usually do _not_ use this C-type decrement, but we do use `-=` .. `tag.closeTagStart -= 1` which is a bit more to type, but not ambiguous